src/utils/helpers.js

```diff
@@ -29,3 +29,59 @@ export const createIdsSelector = createSelectorCreator(memoizeResult);
 
 // Use this selector when you want a shallow comparison of the arguments and you don't need to memoize the result
 export const createShallowSelector = createSelectorCreator(defaultMemoize, shallowEqual);
+
+// isMinimumServerVersion will return true if currentVersion is equal to higher or than the
+// the provided minimum version. A non-equal major version will ignore minor and dot
+// versions, and a non-equal minor version will ignore dot version.
+// currentVersion is a string, e.g '4.6.0'
+// minMajorVersion, minMinorVersion, minDotVersion are integers
+export function isMinimumServerVersion(currentVersion, minMajorVersion = 0, minMinorVersion = 0, minDotVersion = 0) {
+    if (!currentVersion || typeof currentVersion !== 'string') {
+        return false;
+    }
+
+    let firstDotIndex = currentVersion.indexOf('.');
+    if (firstDotIndex < 0) {
```

Comment:
I am curious why you don't use `=== -1` here. It is more explicit than <0, although they are pretty much the same thing. I am just used to seeing `=== -1`

Reply:
you could split based on '.' and take the length of the resulting array...? Could be less code?  
    currentVersion.split('.') // ["4","6","0"]

Reply:
You can decide, I simply feel like it would be less code and easier to read. Plus you wouldn't have to slice later.

Reply:
Good suggestion, that's probably a better way to do it

Reply:
Quick reply here, i’m not reviewing the code but instead of indexOf you can use **includes**

Reply:
I changed it completely anyways but I actually needed the indexes in my original implementation